controllers/rsyncnodedatamovement_controller.go

```diff
@@ -111,21 +108,7 @@ func (r *RsyncNodeDataMovementReconciler) Reconcile(ctx context.Context, req ctr
 		arguments = append(arguments, "--dry-run")
 	}
 
-	// Find the source path for this request. This could be a translation from the compute-local
-	// path (if initiator is a compute node); otherwise it is the rabbit-local path.
-	source, err := r.getSourcePath(ctx, rsyncNode)
-	if err != nil {
-		rsyncNode.Status.EndTime = metav1.Now()
-		rsyncNode.Status.State = nnfv1alpha1.DataMovementConditionTypeFinished
-		rsyncNode.Status.Status = nnfv1alpha1.DataMovementConditionReasonInvalid
-		rsyncNode.Status.Message = err.Error()
-		if err := r.Status().Update(ctx, rsyncNode); err != nil {
-			log.Error(err, "failed to set rsync node as invalid")
-			return ctrl.Result{}, err
-		}
-		return ctrl.Result{}, nil
-	}
-
+	source := rsyncNode.Spec.Source
```

Comment:
We can still lookup the rabbit's clientmount to make sure it has one that matches this path. (This kind of verification is a common theme in the current security coding classes...)

Reply:
done